insecure/attackNetwork.go

```diff
@@ -10,4 +10,8 @@ type AttackNetwork interface {
 	component.Component
 	// Send enforces dissemination of given event via its encapsulated corrupted node networking layer through the Flow network.
 	Send(*Event) error
+	// Observe is the inbound message handler of the attack network.
+	// Instead of dispatching their messages to the networking layer of Flow, the conduits of corrupted nodes
+	// dispatch the outgoing to the attack network through a remote call to this method.
```

Comment:
nit: dispatch the outgoing **message** to the attack network

Reply:
https://github.com/onflow/flow-go/pull/2599/commits/8d8584fa08145a1b1b6f661707397366df80b1c4